src/components/AlertCard/styles.module.css

```diff
@@ -28,40 +32,60 @@
   margin-right: 0.5rem;
 }
 
+.flexGrow {
+  flex-grow: 2;
+  background-color: white;
+}
 .alertColumn > p,
 .subtitleText {
   margin-bottom: 0;
 }
 
 .spacerColumn {
   flex-grow: 2;
-  height: 65px;
-  background-color: #f0f0f0;
+  height: 4.0625rem;
 }
 
 .rightColumn {
   flex-grow: 2;
-  height: 65px;
+  height: 4.0625rem;
   text-align: right;
-  top: -65px;
-  margin-left: 850px;
-  padding-left: 150px;
+  top: -4.0625rem;
+  margin-left: 53.125rem;
+  padding-left: 9.375rem;
 }
 
 .rectangle {
   display: inline-block;
-  height: 70px;
-  width: 8px;
-  background-color: red;
-  padding-left: 5px;
+  height: 4.375rem;
+  width: 0.5rem;
+  padding-left: 0.3125rem;
+}
+
+.flexCollapse {
+  display: flex;
+}
+
+.childCollapse {
+  margin-right: 0.9375rem;
 }
 
 .low {
-  color: alertYellow;
+  width: 2rem;
+  height: 1.5625rem;
+  background: url("/static/medium.svg");
+  background-size: 2.8125rem 3.125rem;
+  position: relative;
+  bottom: 0.8rem;
 }
 
 .medium {
-  color: alertOrange;
+  width: 2rem;
+  height: 1.5625rem;
+  background: url("/static/moderate.svg");
```

Comment:
These should change to the other method of import too

Reply:
fixed